tests/testthat/test-add_ci.tbl_summary.R

```diff
@@ -940,3 +940,100 @@ test_that("add_ci() correctly handles columns of all NAs", {
     "NA%, NA%"
   )
 })
+
+test_that("theme element add_ci.tbl_summary-arg:method", {
+  # test default values
+  expect_silent(
+    tbl <-
+      trial |>
+      select(
+        age = age,
+        age_t.test = age,
+        grade = grade,
+        grade_wilson = grade
+      ) %>%
+      tbl_summary(
+        missing = "no",
+        statistic = list(all_continuous() ~ "{mean}", all_categorical() ~ "{p}%")
+      ) |>
+      add_ci(
+        method = list(
+          ends_with("t.test") ~ "t.test",
+          ends_with("wilson") ~ "wilson"
+        ),
+        style_fun =
+          list(all_continuous() ~ label_style_sigfig(digits = 4),
+               all_categorical() ~ label_style_sigfig(digits = 4, scale =  100))
+      )
+  )
+
+  expect_equal(
+    tbl$table_body |>
+      dplyr::filter(variable == "age") |>
+      dplyr::pull(ci_stat_0),
+    tbl$table_body |>
+      dplyr::filter(variable == "age_t.test") |>
+      dplyr::pull(ci_stat_0)
+  )
+
+  expect_equal(
+    tbl$table_body |>
+      dplyr::filter(variable == "grade") |>
+      dplyr::pull(ci_stat_0),
+    tbl$table_body |>
+      dplyr::filter(variable == "grade_wilson") |>
+      dplyr::pull(ci_stat_0)
+  )
+
+  # changing theme
```

Comment:
Let's move the theme tests to `test-theme_elements_gtsummary.R`. Also there is a function called `with_gtsummary_theme()` that we use for testing, so we don't need to set the themes (in case we forget to reset later).

Reply:
Will do tomorrow. Thanks for the feedback.